Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -695,12 +695,6 @@ Below are procedures for checking the binary security features described above.
     [SNIP]
     ```
 
-#### With idb
-
-[IDB](0x08-Testing-Tools.md#idb) automates the processes of checking for stack canary and PIE support. Select the target binary in the IDB GUI and click the "Analyze Binary…" button.
-
-<img src="Images/Chapters/0x06i/idb.png" alt="IDB Analyze Binary" width="350px" />
```

Comment:
please also delete the image :)

Reply:
here's the path to the image to be removed. See the deleted line.

Reply:
I apologize for the disturbing, I was not really awaken 😖 
As the folder will become empty do you want that I add a `.gitkeep` file?

Reply:
it's not necessary, thanks 😊